src/applications/proxy-rewrite/tests/health.integration.spec.js

```diff
@@ -0,0 +1,50 @@
+const puppeteer = require('puppeteer');
+
+const fullScreen = { width: 1024, height: 800 };
+const mobileScreen = { width: 400, height: 700 };
+
+const VAGOV_HEALTH_URL = 'https://staging.va.gov/health/';
+
+const HEADER = 'header';
+const FOOTER = 'footer';
+
+describe('jest-image-snapshot usage with an image received from puppeteer', () => {
+  let browser;
+  let page;
+  jest.setTimeout(20000);
+
+  const compareScreenshots = async selector => {
+    const element = await page.$(selector);
+    await page.waitFor(1000);
+    const image = await element.screenshot();
+
+    expect(image).toMatchImageSnapshot();
+  };
+  beforeAll(async () => {
+    browser = await puppeteer.launch();
+  });
+  beforeEach(async () => {
+    page = await browser.newPage();
```

Comment:
This might end up being a bottleneck if we write more tests for a particular page. We could probably do more than one test without reloading the page (and actually might benefit from some tests that do multiple things without reloading)

Reply:
That's a really good point. Hopefully when we get to that point the suite can be broken into logical pieces: one `describe` for a certain page at a certain size, for example.

Reply:
Good idea, we can use the screenshot comparison similar to how aXe checks are performed in the e2e tests. Reorganized.